### PR TITLE
Cultures/russia

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: CK2Plus_expanded/history/provinces/1639 - Bacau.txt [hunk 1]
```diff
@@ -21,13 +21,13 @@ religion = slavic_pagan
...
 867.1.1 = {
-	culture = russian
+	culture = ruthenian
 	b_bacau = castle
 	b_targu_neamnt = city
 	b_neamnt = temple
 }
-988.1.1 = { religion = orthodox }
-
+900.1.1 = { religion = orthodox }
```
Comment (schwarherz) [inline CK2Plus_expanded/history/provinces/1639 - Bacau.txt]:
```suggestion
900.1.1 = {
	religion = orthodox
	culture = romanian
}
```

File: CK2Plus_expanded/history/provinces/1639 - Bacau.txt [hunk 2]
```diff
@@ -21,13 +21,13 @@ religion = slavic_pagan
...
+900.1.1 = { culture = romanian }
```
Comment (schwarherz) [inline CK2Plus_expanded/history/provinces/1639 - Bacau.txt]:
```suggestion

```

**Review by schwarherz — CHANGES_REQUESTED** [review]
I'd suggest keeping multiple changes in the same year in the same history entry just for simplicity sake.

File: CK2Plus_expanded/history/provinces/1639 - Bacau.txt (hunk 2 above)
Comment (daniloy) [reply CK2Plus_expanded/history/provinces/1639 - Bacau.txt]: dokay chief

**Review by schwarherz — APPROVED** [review]

**Pull request merged**